include/qore/intern/QoreOperatorNode.h

```diff
@@ -59,6 +60,14 @@ class QoreOperatorNode : public ParseNode {
       ignoreReturnValueImpl();
    }
 
+   DLLLOCAL virtual bool getInParenthesis() const {
```

Comment:
well, yes ;-) I was thinking about it, but finally I decided to make them virtual since `in_parenthesis` member is protected (so children can possibly override these methods).
I'll make `in_parenthesis`->`in_parentheses` private and remove the virtuality...